src/Model/Tag.php

```diff
@@ -325,6 +325,32 @@ private static function getTypeForHash(string $hash)
 		}
 	}
 
+	/**
+	 * Create implicit mentions for a given post
+	 *
+	 * @param integer $uri_id
+	 * @param integer $parent_uri_id
+	 */
+	public static function createImplicitMentions(int $uri_id, int $parent_uri_id)
+	{
+		if ($uri_id == $parent_uri_id) {
```

Comment:
Please explicit the reason for this check.

Reply:
The starting post (this is the only situation where these two values are equal) can't have a mention, so we don't need the further processing there.